github/repos_releases.go

```diff
@@ -253,8 +253,7 @@ func (s *RepositoriesService) DownloadReleaseAsset(ctx context.Context, owner, r
 	}
 
 	// TODO: remove custom Accept header when APIs fully launch.
```

Comment:
Remove this comment, since it's no longer applicable. Also remove the blank line above it, there's no need to separate the `req.Header.Set` call from the code above.

Reply:
I did not care about the comment. I'll do it.